doc/api/stream.md

```diff
@@ -911,10 +911,11 @@ added: v0.9.4
 * `encoding` {String} The encoding to use.
 * Returns: `this`
 
-The `readable.setEncoding()` method sets the default character encoding for
-data read from the Readable stream.
+The `readable.setEncoding()` method sets the character encoding for
+data read from the Readable stream. 
 
-Setting an encoding causes the stream data
+By default, no encoding is assigned and stream data will be returned as 
+Buffer objects.  Setting an encoding causes the stream data
```

Comment:
Double space after the period should be a single space (IMHO and to match the surrounding text).